bql/planner/tracer/trace.go

```diff
@@ -33,7 +38,7 @@ func init() {
 
 	go func() {
 		for e := range c {
-			for _, msg := range e.msgs() {
+			for _, msg := range e.tracerArgs().Msgs {
 				e.w.Write([]byte("["))
 				e.w.Write([]byte(e.t.Format("2006-01-02T15:04:05.999999-07:00")))
```

Comment:
Is there a way to format time that does not use a literal time format?
This reads odd, if a format name was being passed it would be more explicit what is happening.

Reply:
I agree with you.

I have just refactored it to pass the time format name `time.RFC3339Nano`, and also tested it to verify if the timestamps printed by the tracer were indeed in the same expected format as before.